docs/examples/spatial_transformation/histogram_matching.jl

```diff
@@ -0,0 +1,77 @@
+# ---
+# cover: assets/chelsea.png
+# title: Histogram Matching
+# author: Ashwani Rathee
+# date: 2020-11-21
+# ---
+
+# This demo demonstrates the feature of histogram matching.
+
+# In histogram matching,the pixels of source image is manipulated so that the histogram 
+# of reference image matches with source image's histogram.
+
+# Here,histogram matching is shown for gray scale image.But,it can also be applied on 
+# RGB images too.Histogram matching can be used to normalize two images, when the images 
+# were acquired at the same local illumination (such as shadows) over the same location,
+# but by different sensors,atmospheric conditions or global illumination. 
```

Comment:
```suggestion
# Here, histogram matching is shown for the gray scale image. But, it can also be applied on 
# RGB images too. Histogram matching can be used to normalize two images where the images 
# are acquired at the same local illumination (such as shadows) over the same location
# but by different sensors, atmospheric conditions, or global illumination. 
```